### PR TITLE
Update schema and codegen from it

### LINKED ISSUE #15: Migrate to new APIs

### PR DESCRIPTION
**Pull request by s-ylide**
closes #15

- Update schema files (ProjectNext -> ProjectV2)
  - some behaviors (though unavailable for now) may have changed because ProjectV2 field value does not have canonical translation to/from string and I do not know old correspondence.
- Code generation by `graphql_client_codegen`

### CODE REVIEW

File: build.rs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    generate_module_token_stream, CodegenMode, GraphQLClientCodegenOptions,
+};
+use syn::Token;
+
+fn main() {
+    let schema_path = format!("{}/schema.docs.graphql", env::var("OUT_DIR").unwrap());
+    let schema_path = Path::new(&schema_path);
+    if !schema_path.exists() {
+        File::create(schema_path).unwrap();
+        let body = reqwest::blocking::get("https://docs.github.com/public/schema.docs.graphql")
```
Comment (KOBA789) [inline build.rs]: I think that a pre-downloaded and committed schema file is preferable for reproducible builds.
Would you give me your thoughts?

File: src/list_items.graphql [hunk 2]
```diff
@@ -1,73 +1,183 @@
...
               repository {
-                owner {
-                  login
-                }
-                name
+                nameWithOwner
               }
               number
-              labels(first: 100) {
+              labels(first: 10) {
```
Comment (KOBA789) [inline src/list_items.graphql:20]: Why is it decreased? (is it hard-limitted to 10?)

**Review by KOBA789 — COMMENTED** [review]
First of all, *super super* thanks for your contribution!

I have some questions. Would you tell me your thoughts?

File: src/list_items.graphql (hunk 2 above)
Comment (s-ylide) [reply src/list_items.graphql:20]: the old version will search up to about 1000000 nodes and will hit resource limitation:
https://docs.github.com/en/graphql/overview/resource-limitations#node-limit

Comment (KOBA789) [reply src/list_items.graphql:20]: I got it!

File: build.rs (hunk 1 above)
Comment (s-ylide) [reply build.rs]: The file is large with about 60000 lines and updated often, I followed the way `octokit/graphql-schema` does:
https://github.com/octokit/graphql-schema

Comment (KOBA789) [reply build.rs]: > updated often

If so, committing the schema is even so more preferable I think.
Unexpected schema changes may break our builds(eg. type-mismatch), but if the schema is baked, we can diff them and figure out a cause.

I guess `octokit/graphql-schema` is a simple binding library and aims to follow the latest schema, so the codegen is ran in build automatically.

File: src/storage.rs [hunk 3]
```diff
@@ -840,65 +912,89 @@ impl StoreMut<String> for ProjectNextStorage {
...
+                                    }
                                 } else {
                                     return Err((
                                         self,
                                         GlueSQLError::Value(ValueError::ImpossibleCast),
                                     ));
                                 }
                             }
+                            FieldKind::Iteration { .. } => {
+                                let new_str: String = new_value.into();
```
Comment (KOBA789) [inline src/storage.rs:987]: I want Iteration title to id conversion for compatibility and convinience.
Could you implement it?

Comment (s-ylide) [reply src/storage.rs:987]: implemented in `adcdd69`!

File: build.rs (hunk 1 above)
Comment (s-ylide) [reply build.rs]: I understand. `2c5fa2c` added them

File: src/storage.rs (hunk 3 above)
Comment (KOBA789) [reply src/storage.rs:987]: 𝑺𝑼𝑷𝑹𝑬𝑴𝑬

**Review by KOBA789 — APPROVED** [review]
Literary perfect!
And again, great thanks for your work!!

**Pull request merged**